TERIMLERSOZLUGU.md

```diff
@@ -59,10 +59,10 @@ Terimler sözlüğü ayrıca Türkçe'ye çevirilmeyecek sözcüklerin de belirl
 
 
 #F
- * Framework: Framework
+ * Framework: Yapı

```

Comment:
Burda'da çatı olarak çevrilmişti, yapı daga uygun geldi bana onun için http://www.bilisimsozlugu.net/framework